frontend/templates/product-list/sections/FilterableProductsSection/index.tsx

```diff
@@ -119,7 +119,11 @@ export function FilterableProductsSection({
                   overflow="auto"
                   display={{
                      base: 'none',
-                     md: hasAnyVisibleFacet ? 'block' : 'none',
+                     md:
+                        hasAnyVisibleFacet ||
+                        currentRefinements.items.length > 0
```

Comment:
I might turn this expression into a named variable to be sure the logic is the same between the two usages.